internal/commands/all/all.go

```diff
@@ -127,6 +127,7 @@ func BuildCommands(rootCmd *cobra.Command, conf *config.Config) {
 	// Kubernetes
 	kubernetesCommand := commands.BuildCommand(kubernetes.BaseKubernetesCommand(), rootCmd, conf)
 	commands.BuildCommand(kubernetes.CreateCommand(), kubernetesCommand.Cobra(), conf)
+	commands.BuildCommand(kubernetes.ConfigCommand(), kubernetesCommand.Cobra(), conf)
```

Comment:
Would it make sense to have this as `kubeconfig show` and have additional `kubeconfig add` and `kubeconfig remove` commands to add and remove clusters from users default kubeconfig (`~/.kube/config`)?

Reply:
We could think about that for the next release! Good idea.